README.md

```diff
@@ -1,146 +1,129 @@
 
-Create a `index.*.js` in this folder for your platform of choice: `ios`, `android`, or `web`.
 
-These files will load the components to be tested into the Fructose app.
+### Running
+From the root of the project run `yarn` and then either...
 
-Register the component to the name that your app binary expects.
+`yarn e2e:test:web`
 
-```
-import { AppRegistry } from "react-native";
-import Fructose from "fructose";
-import { loadStories } from './components';
+`yarn e2e:test:ios`
 
-AppRegistry.registerComponent("e2eTests", () => Fructose(loadStories));
-```
+`yarn e2e:test:android`
 
-We recommend looking at the examples for [web](./e2eTests/.fructose/index.web.js) and [native](./e2eTests/.fructose/index.ios.js) as there are slight differences.
 
-### Set up the tests
+You will need a (simulator | emulator | chrome) to run these examples.
+### Understanding
 
-Add a setup file in `.fructose`. This is a file that should run once before any of your tests run.
+In the `e2e Test` folder you will find examples of how to consume fructose.
+We suggest first looking at the `scripts` folder to understand the steps required for running tests with Fructose.
 
-In the below examples we will provide example config for jest. If you use a different test runner then you can do the equivalent for that runner.
+Then look at the `examples` folder. 
 
-In this file you will need to run test hooks.
+The `.showcase` file is a description of how a component should be rendered.
+`showcase` files came around because originally fructose consumed `storybook` files. This eventaully resulted in numerous issues so we created an abstraction layer which decouples components from storybook.
+You can see a working example of how showcase to storybook works here.
 
-As a minimum you will need to do the following for mobile:
+[Times-Components-storybook]
 
-```
-import fructose from "@times-components/fructose/setup";
+Showcase files are then loaded into the app index via react-native-showcase-loader function.
 
-beforeAll( async () => {
-  await fructose.hooks.mobile.setup();
-});
 
-afterAll( async () => {
-  await fructose.hooks.mobile.cleanup();
-});
-```
-And for web:
-```
-import fructose from "@times-components/fructose/setup";
+The test file is an example of how to load a component ready for grey or black box testing.
 
-beforeAll( async () => {
-  await fructose.hooks.web.setup();
-});
 
-afterAll( async () => {
-  await fructose.hooks.mobile.cleanup();
-});
-```
+Finally the fructose folder is where all of the different platform indexes live.
 
-You will need to run this setup file at the beginning of your test run. For example, if you are using jest add this to your package.json:
-```
-"jest": {
-  "setupTestFrameworkScriptFile": "./.fructose/setup.js"
-}
-```
 
-The exact structure will depend on the test runner you are using and the platform you are targetting.
+Below are examples of Ios and Web e2eTests running.
```

Comment:
Below are examples of Ios and Web e2eTests running:

Reply:
you makers are pedantic, but I can do better! - 'iOS and web'